4_Marketing-Support/support.md

```diff
@@ -67,6 +67,10 @@ Pre-prepared replies to most common questions are available on the Intercom user
 
 If you have the need to create a new canned reply, you can do so by clicking on the ribbon icon found in the bottom part of the reply box, and selecting `Create`. Managing canned replies can be done by clicking on the `Manage` option.
 
+### Tagging Tickets
+
+Support tickets should be tagged so that we can keep track of common issues. You can tag a ticket by clicking the tag button next to the messages received from the user. We generally only tag the top message, but if the topic changes or there are additional issues you should tag those messages as well.
```

Comment:
@nkk0 The entry is generic enough and not project specific so we can add this here. Just rephrase the first sentence.

Maybe `Support tickets can be tagged to keep track of common issues` or something.